language/move-prover/stackless-bytecode-generator/src/stackless_bytecode.rs

```diff
@@ -102,6 +102,10 @@ pub enum Operation {
     BorrowField(ModuleId, StructId, Vec<Type>, usize),
     BorrowGlobal(ModuleId, StructId, Vec<Type>),
 
+    // Virtual instructions
```

Comment:
Well, all those instructions are virtual, aren't they?

Reply:
Removed the comment.